### PR TITLE
feat: migrate to prettier 3.0.0

### PR DESCRIPTION
**Pull request by tlakomy**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "env": {
     "es6": true,
-    "node": true,
-    "jest": true
```
Comment (WojciechMatuszewskiStedi) [inline .eslintrc.json:5]: Why did we remove the "jest" env here?

File: .eslintrc.json [hunk 2]
```diff
@@ -11,20 +10,17 @@
...
   "parser": "@typescript-eslint/parser",
   "plugins": [
-    "@typescript-eslint",
-    "jest"
+    "@typescript-eslint"
   ],
   "extends": [
     "eslint:recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:jest/recommended"
```
Comment (WojciechMatuszewskiStedi) [inline .eslintrc.json:20]: It seems like we removed jest plugin. Why?

File: .eslintrc.json (hunk 1 above)
Comment (tlakomy) [reply .eslintrc.json]: This was unintentional, reverted that change

File: .eslintrc.json (hunk 2 above)
Comment (tlakomy) [reply .eslintrc.json:20]: See https://github.com/jest-community/eslint-plugin-jest/issues/1398 for reference. We're not actively adding unit tests for this package so I concluded that it's not worth the investment right now. I'd also rather not use `--force` nor `--legacy-peer-deps`.

Comment (WojciechMatuszewskiStedi) [reply .eslintrc.json:20]: That is very valuable context. Thank you.

**Pull request merged**